src/System.Runtime.Numerics/tests/BigInteger/ToByteArray.cs

```diff
@@ -45,20 +45,23 @@ private static IEnumerable<object[]> FromIntTests_MemberDataSeed()
         public void ToByteArray_FromIntTests(int i, bool isUnsigned, bool isBigEndian, byte[] expectedBytes)
         {
             BigInteger bi = new BigInteger(i);
+            byte[] bytes;
 
+#if netcoreapp
```

Comment:
My preference is still to maintain separate tests for the new methods, and to have those tests in separate files.